tests/warn/i22795.check

```diff
@@ -0,0 +1,12 @@
+
+-- Deprecation Warning: tests/warn/i22795/test_1.scala:4:7 -------------------------------------------------------------
+4 |  lib.m() // warn
+  |  ^^^^^^^
+  |  object A is deprecated
+  |---------------------------------------------------------------------------------------------------------------------
+  |Inline stack trace
+  |- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  |This location contains code that was inlined from test_1.scala:5
```

Comment:
the code was inlined from `lib_0.scala`, not `test_1.scala`, or?

Reply:
yes, that was the improvement (to do) I mention in the comment. I hoped to book the progress before opening the can of worms. A can of worms has a long shelf life and can be opened at any time.

Reply:
note this is the (wrong) span it currently reports at.